src/commands/run.js

```diff
@@ -65,13 +67,21 @@ async function executeOnSasViya(filePath, buildTarget, linesToExecute) {
       throw e
     })
 
-  const executionResult = await sasjs.executeScriptSASViya(
-    path.basename(filePath),
-    linesToExecute,
-    contextName,
-    accessToken,
-    executionSession.id
-  )
+  const executionResult = await sasjs
+    .executeScriptSASViya(
+      path.basename(filePath),
+      linesToExecute,
+      contextName,
+      accessToken,
+      executionSession.id
+    )
+    .catch(async (err) => {
+      let log = err.log
+
+      await createOutputFile(log)
+
+      throw new ErrorResponse('Find more error details in the log file.')
```

Comment:
It'd be good if we could output the path to the log file here.

Reply:
`createOutputFile()` will print the path